app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(files_name: str) -> None:
+    command_line = files_name.split(" ")
```

Comment:
You can simply call the `split()` method without any arguments. By default, it will split the string on whitespace